src/shared/util.js

```diff
@@ -919,7 +928,11 @@ function stringToPDFString(str) {
       try {
         const decoder = new TextDecoder(encoding, { fatal: true });
         const buffer = stringToBytes(str);
-        return decoder.decode(buffer);
+        const decoded = decoder.decode(buffer);
+        if (!decoded.includes("\x1B")) {
+          return decoded;
+        }
+        return decoded.replaceAll(/\x1b[^\x1b]*(\x1b|$)/g, "");
```

Comment:
I remember reading somewhere, possibly in some older Bugzilla entry, that regular expression performance may be improved by avoiding capturing groups when they're not actually needed.
```suggestion
        return decoded.replaceAll(/\x1b[^\x1b]*(?:\x1b|$)/g, "");
```